dot/services_integration_test.go

```diff
@@ -726,6 +726,8 @@ func TestNewWebSocketServer(t *testing.T) {
 	cfg.Init.Genesis = genFile
 	cfg.RPC.External = false
 	cfg.RPC.WS = true
+	cfg.RPC.Port = 9545
```

Comment:
Note for reviewer, ports were already in use in the CI causing this test to fail. Changed to bind to different port, but let me know if there is a better solution